tests/e2e/mod.rs

```diff
@@ -41,7 +37,7 @@ async fn get_spans(f: impl FnOnce()) -> Option<Vec<Span>> {
 
 #[cfg(feature = "traces")]
 #[tokio::test(flavor = "multi_thread")]
-async fn it_sends_traces_to_jaeger() {
+async fn traces_are_sent_to_datadog() {
     let log_message = "hello it_sends_traces_to_jaeger";
```

Comment:
This should be updated too probably...?

Reply:
Doesn't really matter, but done for consistency